lib/virtmanager.pm

```diff
@@ -4,216 +4,175 @@ use strict;
+            send_key 'tab';
             type_string $vnet->{ipv6}{dhcpv6}{end};
         }
-        send_key "tab";
-        if ($vnet->{ipv4}{staticrouteipv6}{active} eq "true") {
-            wait_screen_change { send_key "spc" };
-            send_key "tab";
+        send_key 'tab';
+        if ($vnet->{ipv4}{staticrouteipv6}{active} eq 'true') {
+            wait_screen_change { send_key 'spc' };
+            send_key 'tab';
             type_string $vnet->{ipv6}{staticrouteipv6}{tonet};
-            send_key "tab";
+            send_key 'tab';
             type_string $vnet->{ipv6}{staticrouteipv6}{viagw};
         }
     }
     save_screenshot;
-    wait_screen_change { send_key "alt-f" };
+    wait_screen_change { send_key 'alt-f' };
     # step 4
-    if ($vnet->{vnet}{isolatedvnet}{active} eq "true") {
-        send_key "alt-i";
-        send_key "tab";
-    }
-    elsif ($vnet->{vnet}{fwdphysical}{active} eq "true") {
-        wait_screen_change { send_key "tab" };
-        wait_screen_change { send_key "down" };
-        wait_screen_change { send_key "tab" };
+    if ($vnet->{vnet}{isolatedvnet}{active} eq 'true') {
+        send_key 'alt-i';
+        send_key 'tab';
+    }
+    elsif ($vnet->{vnet}{fwdphysical}{active} eq 'true') {
+        wait_screen_change { send_key 'tab' };
+        wait_screen_change { send_key 'down' };
+        wait_screen_change { send_key 'tab' };
         # we only support ANY now
-        if ($vnet->{vnet}{fwdphysical}{destination} eq "any") {
-            # sending tab will go to "mode"
-            send_key "tab";
+        if ($vnet->{vnet}{fwdphysical}{destination} eq 'any') {
+            # sending tab will go to 'mode'
+            send_key 'tab';
         }
-        if ($vnet->{vnet}{fwdphysical}{mode} eq "nat") {
-            send_key "tab";
+        if ($vnet->{vnet}{fwdphysical}{mode} eq 'nat') {
+            send_key 'tab';
         }
-        elsif ($vnet->{vnet}{fwdphysical}{mode} eq "routed") {
-            wait_screen_change { send_key "down" };
+        elsif ($vnet->{vnet}{fwdphysical}{mode} eq 'routed') {
+            wait_screen_change { send_key 'down' };
         }
     }
     # go to enable ipv6 routing
-    send_key "tab";
-    if ($vnet->{vnet}{ipv6routing} eq "true") {
-        send_key "spc";
+    send_key 'tab';
+    if ($vnet->{vnet}{ipv6routing} eq 'true') {
+        send_key 'spc';
     }
     save_screenshot;
     # go to next
-    wait_screen_change { send_key "tab" };
-    if ($vnet->{vnetl}{DNSdomainname} ne "") {
+    wait_screen_change { send_key 'tab' };
+    if ($vnet->{vnetl}{DNSdomainname} ne '') {
         type_string $vnet->{vnet}{DNSdomainname};
     }
     save_screenshot;
     # finish
     valid_step();
     # always close this windows to restart from scratch
-    send_key "ctrl-w";
+    send_key 'ctrl-w';
 }
 
 sub loop_down {
     my ($count) = @_;
-    for (1 .. $count) { send_key "down"; }
+    send_key 'down' for (1 .. $count);
```

Comment:
not really against , but just curious what the reason for this change ?

Reply:
less characters to type ;)

Reply:
hard to argue :)